CMakeSettings.json

```diff
@@ -0,0 +1,51 @@
+﻿{
+    "configurations": [
```

Comment:
This is configuration file generated for Visual Studio - CMake GUI support. Dev does not need to use command line. I will describe this process in a separate doc. You can build from GUI and you can run all Google Test from GUI, visually showing the results with Visual Studio Test Explorer - Google test plugin for that.

Reply:
For what use cases does this file make sense? Do those default settings make sense for most VS users?

Reply:
These configurations are identical to default CMake configuration, building SDK with OTLP protocol and Examples. We can add more configurations here. I was also planning to add (in a separate commit) - configuration that builds with Standard Library instead of nostd.. since there is a large of possible configuration, maybe we can discuss what are the configurations we'd like to show here.

How it works - you select one of the ready-to-use configurations to build. And build it. You can also add more configurations, which get added to this file.

We can review this list and tune in a follow-up PR.